<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>550</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>620</width>
      <height>490</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab1">
     <attribute name="title">
      <string>Panel 1</string>
     </attribute>
     <widget class="QPushButton" name="btn_Copy">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>370</y>
        <width>95</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>Copy</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>190</y>
        <width>121</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Danh sách ảnh chọn</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>325</y>
        <width>80</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Định dạng</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>420</y>
        <width>200</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Copyright © by Khoa Nguyen</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>130</y>
        <width>171</width>
        <height>16</height>
       </rect>
      </property>
      <property name="locale">
       <locale language="Vietnamese" country="Vietnam"/>
      </property>
      <property name="text">
       <string>Đặt tên thư mục lưu</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>60</y>
        <width>180</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Đường dẫn tới thư mục gốc</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="m_url">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>90</y>
        <width>450</width>
        <height>30</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_Move">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>370</y>
        <width>95</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>Move</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>320</y>
        <width>120</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_Cancel">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>370</y>
        <width>95</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="plainTextEdit">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>210</y>
        <width>450</width>
        <height>90</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="m_newFolder">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>150</y>
        <width>250</width>
        <height>30</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="browse">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>90</y>
        <width>30</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>...</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>20</y>
        <width>501</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Panel 1 dùng để lọc file từ danh sách file được chọn (trong 1 thư mục gốc chứa nhiều file)</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab2">
     <attribute name="title">
      <string>Panel 2</string>
     </attribute>
     <widget class="QLineEdit" name="m_url_2">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>110</y>
        <width>450</width>
        <height>30</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="browse_2">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>110</y>
        <width>30</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>...</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>80</y>
        <width>151</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Thư mục JPG đã lọc</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="m_url_3">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>190</y>
        <width>450</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>160</y>
        <width>151</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Thư mục RAW cần lọc</string>
      </property>
     </widget>
     <widget class="QPushButton" name="browse_3">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>190</y>
        <width>30</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>...</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>240</y>
        <width>171</width>
        <height>16</height>
       </rect>
      </property>
      <property name="locale">
       <locale language="Vietnamese" country="Vietnam"/>
      </property>
      <property name="text">
       <string>Đặt tên thư mục lưu</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="m_newFolder_2">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>270</y>
        <width>250</width>
        <height>30</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_OK_2">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>340</y>
        <width>95</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>OK</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>420</y>
        <width>201</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Copyright © by Khoa Nguyen</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>20</y>
        <width>451</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Panel 2 dùng để lọc file RAW dựa trên thư mục chứa file JPG</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>650</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Hỗ trợ</string>
    </property>
    <addaction name="actionInfo"/>
   </widget>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionInfo">
   <property name="text">
    <string>Info</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
